tests/Providers/Rng/TestRNGProvider.php

```diff
@@ -8,7 +8,7 @@
 
 class TestRNGProvider implements IRNGProvider
 {
-    public function __construct(private bool $isSecure = false)
+    public function __construct(private readonly bool $isSecure = false)
```

Comment:
potentially unnecessary readonly as previously discussed

Reply:
Adding the readonly modifier is useful so the developers instantly know that it's only set in the constructor and is not modified later on anywhere, not even in its own class.
In this case, I kind of understand it as there is really not much happening here, however, I put it there for consistency's sake. In my opinion, if a variable is only set once and it's never going to get modified later on, it should be prefixed with `readonly`. Not prefixing it could mean that you might be able to modify its value somewhere in its own class.